### DISCUSSION TITLE
Touchscreen useability

### DISCUSSION THREAD

**Discussion by JonnyTech** [doublecmd/doublecmd#1466] [Suggestions]
Navigating the filesystem and executing files is infuriatingly difficult and tedious when using a touchscreen. The list items do not respond very well to touches and many attempts have to be made. Is there a touch mode available?

**Comment by alexx2000** [comment]
No, DC is mouse and keyboard oriented.

**Reply by JonnyTech** [reply]
Understood, thanks. Are there any future plans to make it more touch friendly?

**Reply by JonnyTech** [reply]
FYI: My current workaround is to have a `cm_open`![cm_open](https://github.com/doublecmd/doublecmd/assets/3584732/14a41ef4-df65-425f-9865-413df15f2b08) button on my toolbar then click that to change folder, open archive, execute file, etc. Double tapping would be easier but at least that button works. Is it possible to make the button bigger or more prominent?

**Reply by alexx2000** [reply]
> Are there any future plans to make it more touch friendly?

No, I don't have a such plans currently.

> Is it possible to make the button bigger or more prominent?

One button - no, only all buttons.
